views/Graph.react.js

```diff
@@ -342,15 +425,25 @@ class Graph extends Component {
         let ctx = canvas.getContext('2d')
 
         img.onload = () => {
-          ctx.drawImage(img, 0, 0, canvas.width, canvas.height, 0, 0, canvas.width, canvas.height)
+          ctx.drawImage(
+            img,
+            0,
+            0,
+            canvas.width,
+            canvas.height,
+            0,
+            0,
+            canvas.width,
+            canvas.height
+          )
         }
         img.src = svgUrl
 
         this.setState({
```

Comment:
is this calling setState inside of a setState function?

Reply:
yes, it definitely is a setState inside a setState function.